src/sentry/consumers/dlq.py

```diff
@@ -98,7 +98,13 @@ def submit(self, message: Message[KafkaPayload]) -> None:
         )
 
         if isinstance(message.value, BrokerValue):
-            if message.value.timestamp < min_accepted_timestamp:
+            # Normalize the message timezone to be UTC
+            if message.value.timestamp.tzinfo is None:
+                message_timestamp = message.value.timestamp.replace(tzinfo=timezone.utc)
+            else:
+                message_timestamp = message.value.timestamp.astimezone(timezone.utc)
```

Comment:
just keep the original timezone if there is one, otherwise we're changing the actual time?
```suggestion
            else:
                message_timestamp = message.value.timestamp
```

Reply:
this keeps it, but it just converts it to UTC (if its already not)

Reply:
would you prefer we keep original timezone?

Reply:
i guess the comparison will work if there is an original timezone. We don't have to convert to utc here. Let's only add utc if there is no timezone